src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
Just a small suggestion to remove the code you are not using like the serviceWorker code and file or any other files that create-react-app automatically generates that you don't need